website/static/js/osfHelpers.js

```diff
@@ -280,15 +280,28 @@ var isEmail = function(value) {
   * If `str` is falsy, return {}.
   * Modified from getQueryParameters plugin by Nicholas Ortenzio (MIT Licensed).
   */
-var urlParams = function(str) {
+var urlParams = function(str, isEmail) {
     var stringToParse = str || document.location.search;
     if (!stringToParse) {
         return {};
     }
     return (stringToParse).replace(/(^\?)/,'').split('&')
-        .map(function(n){return n = n.split('='),this[n[0]] = decodeURIComponent(n[1]).replace(/\+/g, ' '),this;}.bind({}))[0];
+        .map(function(n){return n = n.split('='),this[n[0]] = decodeURIComponent(n[1]),this;}.bind({}))[0];
 };
 
+/**
+  * Get email address as an object.
+  * If `str` is falsy, return {}.
+  * Modified from getQueryParameters plugin by Nicholas Ortenzio (MIT Licensed).
+  */
+var urlEmail = function(str) {
```

Comment:
This shouldn't be necessary. You should URI-encode the email addresses in the query parameters:

```js
encodeURIComponent('ßteve@test.com')  // "%C3%9Fteve%40test.com"
```

Then when you read the query param, decode:

```js
decodeURIComponent(urlParams().email)   // 'ßteve@test.com'
```